src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File firstFile = new File(fromFileName);
+        File secondFile = new File(toFileName);
+
+        int buy = 0;
```

Comment:
The variables `buy`, `supply`, and `result` should be local to the try block where they are used. They don't represent the state of the object and are not used outside of the try block, so there is no need to make them fields of the class.